ui/src/router/index.ts

```diff
@@ -15,6 +16,11 @@ const router = createRouter({
       name: 'Node Details',
       component: NodeDetails
     },
+    {
```

Comment:
This is acceptable for now, but as noted above this will eventually become the Skeleton page and then eventually house the entire experience.

Reply:
Okay